### ISSUE TITLE
--version option doesn't work with dotnet format in .NET 6 RC2 SDK

### ISSUE THREAD

**Issue by akoeplinger** [dotnet/format#1384]
```bash
alexander:~$ dotnet format --version
--version option cannot be combined with other arguments.

[... help text ...]
```

This seems to be the same error as in https://github.com/dotnet/format/issues/1206 and https://github.com/dotnet/format/issues/1136, but those were closed with a supposed fix. Someone also noticed this problem in https://github.com/dotnet/format/issues/1268#issuecomment-933844951.

My dotnet SDK details:

```bash
alexander:~$ dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.100-rc.2.21426.20
 Commit:    b1683c7ba7

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  11.0
 OS Platform: Darwin
 RID:         osx.11.0-x64
 Base Path:   /usr/local/share/dotnet/sdk/6.0.100-rc.2.21426.20/

Host (useful for support):
  Version: 6.0.0-rc.2.21423.6
  Commit:  8ddb633523
```

Looking at the dotnet-format.dll in the SDK folder it seems to be built from https://github.com/dotnet/format/commit/727a4ec856a56e39e1dea812107b876199865e32.

/cc @JoeRobich

**Comment by akoeplinger** [comment]
Ah I think this is actually intentional with the move into the dotnet SDK and the breakinng changes to the command line arguments in #1292.

 --version argument might still make sense given that it's likely not just me who was confused if you look at https://github.com/dotnet/format/issues/1268#issuecomment-933844951 and the preceding comments.

**Comment by JoeRobich** [comment]
@akoeplinger  I cannot reproduce this on the released RC2 SDK version 6.0.100-rc.2.21505.57. Can you update to the released version and let me know if this is resolved for you.
